src/features/copy-on-y.js

```diff
@@ -1,14 +1,10 @@
-import copyToClipboard from 'copy-text-to-clipboard';
```

Comment:
This was the last piece of `copy-text-to-clipboard'

Now there's only the disabled `copy-to-markdown`